packages/vda-common/src/contract.ts

```diff
@@ -10,62 +92,68 @@ export type CONTRACT_NAMES =
     "StorageNodeRegistry"
     ;
 
-
-export interface CONTRACT_INFO {
-    abi: any,
-    address: string
-}
-
+/**
+ * @todo Deprecate in favour of `NETWORK_DEFINITIONS`
+ */
 export const CONTRACT_ADDRESS : Record<CONTRACT_NAMES, Record<string, string | null>> = {
```

Comment:
This deprecated `CONTRACT_ADDRESS` still holds info not in the network defintiions, such as for the DID registry addresses.

I gues we should have a similar object for the DID VDA, gathering the registry address, its blockchain and other related information (name registry?).